src/opera/test/data/test_dswx_ni_algorithm_parameters.yaml

```diff
@@ -2,12 +2,17 @@ runconfig:
     name: dswx_ni_workflow_algorithm
 
     processing:
-        # dswx_workflow 'opera_dswx_ni', 'twele', 'opera_dswx_ni_inundated_vegetation'
+        # dswx_workflow 'opera_dswx_ni', 'twele'
         dswx_workflow: 'opera_dswx_ni'
-        # valid values for polarizations
-        # empty for all polarizations found in RSLC
-        # [polarizations] for list of specific frequency(s) e.g. [HH, HV] or [HH]
-        polarizations: ['VV', 'VH']
+        # Polarizations to be used for DSWx-SAR
+        # [polarizations] for list of specific frequency(s) e.g. [VV, VH] or [VV]
+        # 'dual-pol', 'co-pol', 'cross-pol' will search the polarizations Input GeoTiff files have.
```

Comment:
This comment "....search the polarizations Input GeoTiff files have."  is awkward.  Maybe "... 'cross-pol' search the polarization from Input GeoTiff files.

Reply:
Since this file is essentially copied wholesale from an ADT repo (meaning the documentation is theirs), I'd rather not make our own changes so its easier to track actual changes when we copy over newer versions.